lib/Core/SQLiteBuildDB.cpp

```diff
@@ -65,8 +78,6 @@ class SQLiteBuildDB : public BuildDB {
     if (result == SQLITE_ERROR) {
       version = -1;
     } else {
-      assert(result == SQLITE_OK);
```

Comment:
This change loses this `result` code here... we should also error if it is not `SQLITE_OK` at this point right?